contracts/Removal.sol

```diff
@@ -25,7 +25,7 @@ contract Removal is
 
   struct Vintage {
     address supplier;
-    uint16 vintage;
+    uint256 vintage;
```

Comment:
p sure uint256 is just as cheap if not cheaper than other int sizes because of the 256bit wordsize of the evm... a fun fact I recall about solidity that also makes my life easier here